tools/ports/vorbis.py

```diff
@@ -17,22 +17,35 @@ def get(ports, settings, shared):
 
   ports.fetch_project('vorbis', 'https://github.com/emscripten-ports/vorbis/archive/' + TAG + '.zip', 'Vorbis-' + TAG, sha512hash=HASH)
   libname = ports.get_lib_name('libvorbis')
-
-  def create():
-    logging.info('building port: vorbis')
-
-    source_path = os.path.join(ports.get_dir(), 'vorbis', 'Vorbis-' + TAG)
-    dest_path = os.path.join(shared.Cache.get_path('ports-builds'), 'vorbis')
-
-    shutil.rmtree(dest_path, ignore_errors=True)
-    shutil.copytree(source_path, dest_path)
-
-    final = os.path.join(dest_path, libname)
-    ports.build_port(os.path.join(dest_path, 'lib'), final, [os.path.join(dest_path, 'include')],
-                     ['-s', 'USE_OGG=1'], ['psytune', 'barkmel', 'tone', 'misc'])
-    return final
-
-  return [shared.Cache.get(libname, create)]
+  libnamefile = ports.get_lib_name('libvorbisfile')
+  libnameenc = ports.get_lib_name('libvorbisenc')
```

Comment:
in general we use `_` to split out parts of local names (see elsewhere in this file, like `source_path`), please follow those conventions